redhat/apmplanner2-0.patch

```diff
@@ -0,0 +1,63 @@
+diff --git a/apm_planner.pro b/apm_planner.pro
```

Comment:
I've submitted this as a separate pr (#1053).  I think it makes more sense to have this particular change committed to the apm planner sources, rather than carrying it as a patch in the rpm.